components/ExercisesList/ExercisesList.tsx

```diff
@@ -2,62 +2,91 @@ import React from 'react';
 import {Pressable, ScrollView, Text, View} from 'react-native';
 import {exercises, paths} from '../../utils/constants';
 import {useQuery} from 'react-query';
-import {getSubData, getUserId} from '../../utils/firebaseUtils';
-import {ExerciseListItem, Navigation} from '../../types/CommonTypes';
-import {useNavigation, useRoute} from '@react-navigation/native';
+import {getSubArray, getUserId} from '../../utils/firebaseUtils';
+import {ExerciseDetailsForm, Navigation} from '../../types/CommonTypes';
+import {
+  useFocusEffect,
+  useNavigation,
+  useRoute,
+} from '@react-navigation/native';
 import Button from '../Button/Button';
 import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
 import {Colors} from '../../utils/colors';
 import {faPlus} from '@fortawesome/free-solid-svg-icons';
 
+import style from './style';
 import globalStyle from '../../assets/styles/globalStyle';
 
 const ExercisesList: React.FC = () => {
   const navigation = useNavigation<Navigation>();
   const uid = getUserId();
   const route = useRoute();
-  const workoutKey = route.params?.key;
+  const workoutKey = route.params?.workoutKey;
 
-  const getExercises = async () => {
-    const data = await (
-      await getSubData(
-        paths.USERS_PATH,
-        paths.WORKOUTS_PATH + '/' + workoutKey,
-        uid,
-      )
-    ).val();
-    return data.exercises;
-  };
+  const getExercises = async () =>
+    await getSubArray(
+      paths.USERS_PATH,
+      `${paths.WORKOUTS_PATH}/${workoutKey}/${paths.EXERCISES_PATH}`,
+      uid,
+    );
+
+  const {data: savedExercises, refetch} = useQuery<
+    Array<ExerciseDetailsForm>,
+    Error
+  >(exercises.EXERCISES_KEY, getExercises);
 
-  const {data: savedExercises} = useQuery<Array<ExerciseListItem>, Error>(
-    exercises.EXERCISES_KEY,
-    getExercises,
+  useFocusEffect(
+    React.useCallback(() => {
+      refetch();
+    }, [refetch]),
```

Comment:
Some of the hooks are called with their name and others are called with `.` syntax, this looks weird and it seems like 2 different people were working on this. You need consistency.